frontend/tailwind.config.js

```diff
@@ -4,6 +4,7 @@ export default {
     content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
     theme: {
         colors: {
+            'lm-white': '#FFFFFF',
```

Comment:
is this necessary? Since we already have white
Everything else is superb

Reply:
Need to fix that. Thank you